index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Skull-Css</title>
+
+    <style>
+      * {
+        margin: 0;
+        padding: 0;
+        box-sizing: border-box;
+      }
+
+      body {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+
+        height: 100vh;
+
+        width: 100vw;
+
+        background: #000000;
+
+        transform: scale(0.85);
+
+        overflow: hidden;
+      }
+
+      .skull {
+        width: 150px;
+        height: 100px;
+
+        border-radius: 50px;
+        background: #ffffff;
+
+        position: relative;
+        top: 10px;
+      }
+
+      .skull::after,
+      .skull::before {
+        content: "";
+        width: 49px;
+        height: 39px;
+
+        border-radius: 50%;
+
+        display: inline-block;
+
+        background: #000000;
+      }
+
+      .skull::after {
+        transform: translate(31px, 31px) rotate(40deg);
+      }
+
+      .skull::before {
+        transform: translate(21px, 31px) rotate(-40deg);
+      }
+
+      [class^="teeth"] {
+        width: 15px;
+        height: 20px;
+
+        display: inline-block;
+
+        background: #ffffff;
+
+        position: absolute;
+        bottom: 0;
+      }
+
+      .teeth-1 {
+        transform: translate(-5px, 30px);
+      }
+
+      .teeth-2 {
+        transform: translate(15px, 30px);
+      }
+
+      .teeth-3 {
+        transform: translate(35px, 30px);
+      }
+
+      [class^="bone"] {
```

Comment:
what this ^ represents in this selector

Reply:
this is to target attributes value that has a starting letter of x.